lib/active_model/serializer.rb

```diff
@@ -9,12 +9,12 @@ class Serializer
     autoload :Adapter
     autoload :Lint
     autoload :Associations
+    autoload :Attributes
     include Configuration
     include Associations
+    include Attributes
```

Comment:
If we're going to include it when the file is evaluated, there's no advantage to the autoload, right? Just curious what your thinking here is.